misc/python/materialize/checks/all_checks/alter_table.py

```diff
@@ -0,0 +1,95 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from textwrap import dedent
+
+from materialize.checks.actions import Testdrive
+from materialize.checks.checks import Check
+
+
+class AlterTable(Check):
+    def initialize(self) -> Testdrive:
```

Comment:
You probably need a `def _can_run` to make sure this doesn't run on a version without support